action.yml

```diff
@@ -0,0 +1,38 @@
+name: 'Run quickstart'
+description: 'Runs quickstart docker image'
+inputs:
+  version:
+    description: "Version (docker tag) of quickstart image to use"
+    required: true
+    default: "latest"
+runs:
+  using: "composite"
+  steps:
+    - name: Setup docker (missing on MacOS)
+      shell: bash
+      if: runner.os == 'macos'
```

Comment:
Could we add a step that verifies that the runner is compatible and provide a human friendly error? That'll help prevent cases where someone uses this with macos-latest and can't figure out why it doesn't work. An error should occur on Windows too.